packages/dd-trace/src/appsec/rasp.js

```diff
@@ -3,23 +3,102 @@
 const { storage } = require('../../../datadog-core')
 const web = require('./../plugins/util/web')
 const addresses = require('./addresses')
-const { httpClientRequestStart } = require('./channels')
+const { httpClientRequestStart, setUncaughtExceptionCaptureCallbackStart } = require('./channels')
 const { reportStackTrace } = require('./stack_trace')
 const waf = require('./waf')
+const { getBlockingAction, block } = require('./blocking')
+
+class AbortError extends Error {
```

Comment:
if it is an Error that could be handled by the application, should we use something more descriptive like DatadogRaspAbortError?

Reply:
Currently it is an internal class, they can't do a `instanceof AbortError` because we are not exporting it. But yes, I think that we can change it, especially for `error.name` property.

Reply:
Class modified, resolving the thread